src/graphnet/models/detector/icecube.py

```diff
@@ -40,6 +40,75 @@ def _forward(self, data: Data) -> Data:
 
         return data
 
+class IceCube86_v2(Detector):
+    """`Detector` class for IceCube-86 with nodes as doms."""
+
+    # Implementing abstract class attribute
+    features = FEATURES.ICECUBE86
+
+    @property
+    def nb_inputs(self) -> int:
+        return len(self.features) + 7
```

Comment:
Is this true? Does it add an additional 7 features compared to what's in `FEATURES.ICECUBE86`?

Reply:
Yes. These are the aggregated pulse information.  See https://github.com/icecube/graphnet/blob/b7420b8163c8eb4f30a4dd0b3784582fe4dcb974/src/graphnet/data/sqlite_dataset.py#L132